libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.scss

```diff
@@ -0,0 +1,163 @@
+@import 'libs/sass/mixins';
+
+.product-tiles {
+  @include flexbox();
+  @include flex-direction(row);
+  @include justify-content(space-around);
+  @include flex-wrap(wrap);
+
+  .product-tiles__tile {
+    display: inherit;
+    @include flex-direction(column);
+    @include align-items(center);
+    width: 37.5%;
+    margin: 3rem 2rem;
+
+    .product-tiles_tile__identity {
+      width: 4.5rem;
+      margin-bottom: 1.25rem;
+    }
+  }
+}
+
+#geocoding-graphic {
+  height: 20rem;
+  object-fit: cover;
+}
+
+#map-section {
+  #map-section-description,
+  #map-section-map {
+    @include flexbox();
+    @include flex(1);
+    @include flex-shrink(10);
+  }
+
+  #map-section-description {
+    padding: 10rem 0;
+
+    p {
+      max-width: 50%;
+    }
+  }
+
+  #map-section-map {
+  }
+}
+
+#map-section {
+  width: auto;
+  height: 50rem;
+  display: flex;
+}
+
+#map-section h3 {
+  padding: 1rem;
+}
+
+// toggle button start
+
+/* The switch - the box around the slider */
+.switch {
+  margin-right: 1rem;
+  margin-bottom: 1rem;
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+}
+
+/* Hide default HTML checkbox */
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+/* The slider */
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+.slider:before {
+  position: absolute;
+  content: "";
+  height: 26px;
+  width: 26px;
+  left: 4px;
+  bottom: 4px;
+  background-color: white;
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+input:checked + .slider {
+  background-color: #2196F3;
+}
+
+input:focus + .slider {
+  box-shadow: 0 0 1px #2196F3;
+}
+
+input:checked + .slider:before {
+  -webkit-transform: translateX(26px);
+  -ms-transform: translateX(26px);
+  transform: translateX(26px);
+}
+
+/* Rounded sliders */
+.slider.round {
+  border-radius: 34px;
+}
+
+.slider.round:before {
+  border-radius: 50%;
+}
```

Comment:
This one is a matter of consistency. My intent here is not to prevent you from having any creative freedom, however it's worth considering that these styles are not usable anywhere else in the application/monorepo in their current state. Either pull them out into the main application styles so they can be re-used on the other remaining maps or consider adopting the pre-styled form input components that are used in other parts of this application. I'm also perfectly OK with having a separate "skin" for input by creating a new "view" that extends the current checkbox input with a different set of styles. In this way, the styles would be bound to a checkbox component, and not a consuming component. 

Whatever you want to do here is up to you, but the overall goal should be consistency and as much DRY-ness as possible.

Reply:
I'll remove this and use the checkbox mentioned later here